appliances/mikrotik-chr.gns3a

```diff
@@ -29,17 +29,17 @@
             {
               "filename": "chr-6.45.6.img",
               "version": "6.45",
```

Comment:
Shouldn't the `version` be 6.45.6, to match with the corresponding entry in the `"versions": [ ... ]` section?  Or doesn't it matter?

(Ditto 6.44 -> 6.44.5 a few lines later)

Reply:
@xanio shouldn't we use the full version?

Reply:
Fixed in https://github.com/GNS3/gns3-registry/pull/463